docs/did-button-branding/index.md

```diff
@@ -0,0 +1,133 @@
+---
+layout: "doc"
+title: "Branding DID.app buttons."
+abstract: "DID.app provides several styles of icon that you can use to let people sign in."
+---
+
+<style>
+  .markdown-body img {
+    background: none;
+  }
+</style>
+
+You can use either our "D.I.D Logo" or "D Icon" to indicate to your users that they may sign in with DID.app.
```

Comment:
```suggestion
You can use either our "DID Logo" or "D Icon" in your button.
```

Reply:
I was wondering about giving the "DID" logo a name to separate it from the "D logo". They're all DID.app logos so I thought D.I.D made it specific I was talking about the three letter version

Reply:
Makes sense, why not say 'You can use either our three letter 'DID' logo or single letter 'D' icon.

Reply:
better, thanks